src/operator/elemwise_op_common.h

```diff
@@ -53,8 +53,8 @@ template<int n_in, int n_out>
 inline bool ElemwiseShape(const nnvm::NodeAttrs& attrs,
                           std::vector<TShape> *in_attrs,
                           std::vector<TShape> *out_attrs) {
-  CHECK_EQ(in_attrs->size(), n_in) << " in operator " << attrs.name;
-  CHECK_EQ(out_attrs->size(), n_out) << " in operator " << attrs.name;
+  CHECK_EQ((int)in_attrs->size(), n_in) << " in operator " << attrs.name;
```

Comment:
ok, well what do you suggest? let the warning stand? There's a lot of mismatches between signed and unsigned all over the place. It's not clear if some items, such as template arguments, can be safely switched to size_t instead of 'int'